src/gui/properties/trackerlistwidget.h

```diff
@@ -54,7 +54,7 @@ class TrackerListWidget : public QTreeWidget
         COL_TIER,
         COL_URL,
         COL_STATUS,
-        COL_RECEIVED,
+        COL_CONNECTIONS,
```

Comment:
This was changed because the column name "Received" is confusing. No one knows what it means.

libtorrent refers to this data as "peers," but this is different than what a user normally thinks of as a "peer."

Reply:
IMO `connections` could be wrongly read as on-going connection count, which is not true.

Reply:
Hmm, utorrent doesn't have this field...

Reply:
>Shall we give it some time and see if anyone objects?

lets proceed!